client/packages/purchase_orders/src/DetailView/Footer/StatusChangeButton/utils.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  LocaleKey,
+  PurchaseOrderNodeStatus,
+  SplitButtonOption,
+  TypedTFunction,
+} from '@openmsupply-client/common';
+import { purchaseOrderStatuses, statusTranslation } from '../utils';
+
+export type PurchaseOrderStatusOption =
+  SplitButtonOption<PurchaseOrderNodeStatus>;
+
+export const getStatusOptions = (
+  currentStatus: PurchaseOrderNodeStatus | undefined,
+  getButtonLabel: (status: PurchaseOrderNodeStatus) => string
+): PurchaseOrderStatusOption[] => {
+  if (!currentStatus) return [];
+
+  const options: [
+    PurchaseOrderStatusOption,
+    PurchaseOrderStatusOption,
+    PurchaseOrderStatusOption,
+    PurchaseOrderStatusOption,
+  ] = [
+    {
+      value: PurchaseOrderNodeStatus.New,
+      label: getButtonLabel(PurchaseOrderNodeStatus.New),
+      isDisabled: true,
+    },
+    // TODO: Authorised should only be considered if pref is on?
```

Comment:
I guess can leave this to sync translation mapping from both permissions + authorisation boolean as discussed in standup?